packages/neocmakelsp/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Decodetalkers/neocmakelsp
+TERMUX_PKG_DESCRIPTION="the lsp for cmake"
+TERMUX_PKG_LICENSE=MIT
+TERMUX_PKG_MAINTAINER=@termux
+TERMUX_PKG_VERSION=0.7.1
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE/archive/v$TERMUX_PKG_VERSION.tar.gz
```

Comment:
`$TERMUX_PKG_HOMEPAGE` should not be used as part of the `$TERMUX_PKG_SRCURL`, in case the homepage is changed later.
```suggestion
TERMUX_PKG_SRCURL=https://github.com/Decodetalkers/neocmakelsp/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
```